libraries/AP_HAL_ChibiOS/hwdef/MicoAir743v2/README.md

```diff
@@ -0,0 +1,96 @@
+# MicoAir743v2 Flight Controller
+
+The MicoAir743v2 is a flight controller designed and produced by [MicoAir Tech](http://micoair.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - BMI088/BMI270 dual IMUs
+ - SPL06 barometer
+ - QMC5883L magnetometer
+ - AT7456E OSD
+ - 9V 3A BEC; 5V 3A BEC
+ - MicroSD Card Slot
+ - 8 UARTs
+ - 11 PWM outputs
+ - 1 I2C
+ - 1 SWD
+
+## Physical
+
+![MicoAir H743 v2.0 Physical Size](MicoAir743v2_Physical_Size.jpg)
+
+## UART Mapping
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (MAVLink2, DMA-enabled)
+ - SERIAL2 -> UART2 (DisplayPort, DMA-enabled)
+ - SERIAL3 -> UART3 (GPS, DMA-enabled)
+ - SERIAL4 -> UART4 (MAVLink2, DMA-enabled)
+ - SERIAL5 -> UART5 (User, DMA-enabled)
+ - SERIAL6 -> UART6 (RCIN, DMA-enabled)
+ - SERIAL7 -> UART7 (RX only, ESC Telemetry, DMA-enabled)
+ - SERIAL8 -> UART8 (MAVLink2, DMA-enabled)
+
+## RC Input
+
+The default RC input is configured on the UART6. And RC could be applied instead at a different UART port such as UART1, UART4 or UART5, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL6 _Protocol to something other than '23'.

```

Comment:
this will save me time when I make its wiki page:
```suggestion
The UART6  is compatible with all ArduPilot supported receiver protocols, 

* PPM is not supported.

  *  SBUS/DSM/SRXL connects to the RX6 pin.

  * FPort requires connection to TX6 . See FPort Receivers.

  *  CRSF also requires a TX6 connection, in addition to RX6, and automatically provides telemetry.

  *  SRXL2 requires a connection to TX6 and automatically provides telemetry. Set SERIAL6_OPTIONS to “4”.

Any UART can also be used for RC system connections in ArduPilot and is compatible with all protocols except PPM. See Radio Control Systems for details.
```

Reply:
Got it. Thanks!